code/backend/pages/04_Configuration.py

```diff
@@ -174,7 +184,7 @@ def validate_documents():
     post_answering_prompt_help = "You can configure a post prompt that allows to fact-check or process the answer, given the sources, question and answer. This prompt needs to return `True` or `False`."
     use_on_your_data_format_help = "Whether to use a similar prompt format to Azure OpenAI On Your Data, including separate system and user messages, and a few-shot example."
     post_answering_filter_help = "The message that is returned to the user, when the post-answering prompt returns."
-
+    ai_assistant_type_help = "Whether to use the default user prompt or other."
```

Comment:
Since there are only two options, I would rephrase this to "Whether to use the default user prompt or the Legal Assistance user prompt. Refer to the Legal Assistance README for more details."